Image_Viewer/theme.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+class Theme:
+    def __init__(self):
+        themeDictionary = self.read_theme()
+        self.themes = themeDictionary["themes"]
+        self.current_theme=themeDictionary["lastTheme"]
+        
```

Comment:
Tiny comment: There's a lot of space between the `__init__` and `set_theme` methods, that can be removed - just need a one line space